docs/src/pages/configurations/custom-webpack-config/index.md

```diff
@@ -96,28 +96,30 @@ If so, this is how you do it using the Full Control Mode.
 Add following content to the `webpack.config.js` in your Storybook config directory.
 
 ```js
-const path = require('path');
+const path = require("path");
 
 module.exports = (baseConfig, env, defaultConfig) => {
   // Extend defaultConfig as you need.
 
   // For example, add typescript loader:
   defaultConfig.module.rules.push({
     test: /\.(ts|tsx)$/,
-    include: path.resolve(__dirname, '../src'),
-    loader: require.resolve('ts-loader')
+    include: path.resolve(__dirname, "../src"),
+    loader: require.resolve("ts-loader")
   });
-  defaultConfig.resolve.extensions.push('.ts', '.tsx');
+  defaultConfig.resolve.extensions.push(".ts", ".tsx");
 
   return defaultConfig;
 };
 ```
 
+For full instructions on Typescript setup, check [our dedicated Typescript page](/configurations/typescript-config/).
+
 ## Using Your Existing Config
 
 You may have an existing Webpack config for your project. So, you may need to copy and paste some config items into Storybook's custom Webpack config file.
 
 But you don't need to. There are a few options:
 
--   Simply import your main Webpack config into Storybook's `webpack.config.js` and use the loaders and plugins used in that.
--   Create a new file with common Webpack options and use it in both inside the main Webpack config and inside Storybook's `webpack.config.js`.
+* Simply import your main Webpack config into Storybook's `webpack.config.js` and use the loaders and plugins used in that.
```

Comment:
Can you remove this `Simply?`. If it's so simple, users wouldn't need to read the docs.

It should just say `Import your main Webpack...`

Reply:
hahaha i didnt write that